radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java

```diff
@@ -77,7 +78,7 @@
 		ValidationState validationState = this.newViewState.computeIfAbsent(thisView, k -> validatorSet.newValidationState());
 
 		// check if we have gotten enough new-views to proceed
-		if (!(validationState.addSignature(node, signature) && validationState.complete())) {
+		if (!(validationState.addSignature(node, 0L, UInt256.ZERO, signature) && validationState.complete())) {

```

Comment:
Add comment that new-views here don't require timestamps

Reply:
OK, added.  Reworked the addSignature(...) method a bit, as it doesn't require the signature weight to be passed in.